components/Search/Search.jsx

```diff
@@ -118,7 +118,8 @@ const Search = ({ query, type }) => {
             <>
               <div className="lbh-table-header">
                 <h2 className="govuk-fieldset__legend--m govuk-custom-text-color">
-                  {type.toUpperCase()} SEARCH RESULT
+                  {type === 'cases' ? 'RECORDS' : type.toUpperCase()} SEARCH
```

Comment:
Can we rename the `type` instead?

Reply:
We're using type for the people label so we need to keep it 
![Screenshot 2020-12-16 at 10 28 03](https://user-images.githubusercontent.com/9123541/102336876-acf8fb80-3f89-11eb-8e13-55a613f619cc.png)

Reply:
yes, what I meant is that we are defining `type="cases"` in https://github.com/LBHackney-IT/lbh-social-care/blob/create_new_user/pages/cases/index.jsx#L9. We should make that `records`.